### PR TITLE
New Fact About JS

### PR DESCRIPTION
**Pull request by AndriusAlimas**
Added new Fact about JS!

## Purpose
adding a new fact to the file

## Approach
adds a new fact

## Checklist
_Reviewer to complete_
- [x] Code conforms to coding standards
- [ ] Changes made resolve the problem or meet the acceptance criteria of the feature
- [ ] Able to run the code locally
- [ ] Existing and new tests pass
- [ ] Documentation has been updated

### CODE REVIEW

**Comment by AndriusAlimas** [comment]
How do you like new Fact?

**Comment by AndriusAlimas** [comment]
That is perfect!

File: facts.md [hunk 1]
```diff
@@ -1 +1,2 @@
 - A group of owls is called a parliament of owls
+-The evolution and use of JavaScript, a language developed in 10 days back in 1995!
```
Comment (rachel-emsley) [inline facts.md]: Great work.
```suggestion
- The evolution and use of JavaScript, a language developed in 10 days back in 1995!
```

**Review by rachel-emsley — CHANGES_REQUESTED** [review]
Just a small typo, but great work

**Comment by Tadas89** [comment]
I like comment section :)

**Pull request merged**